sycl/unittests/queue/wait.cpp

```diff
@@ -87,6 +95,7 @@ TEST(QueueWaitTest, USMEventClear) {
   Q.memset(HostAlloc, 42, 1);
   Q.wait();
   ASSERT_EQ(TestContext->NEventsWaitedFor, 1);
+  ASSERT_EQ(TestContext->EventReferenceCount, 0);
   Q.wait();
   ASSERT_EQ(TestContext->NEventsWaitedFor, 1);
```

Comment:
The event reference count should be nil after the second call to wait() also, isn't it?

Reply:
Right, I just don't think it's worth checking once it already hits 0.

Reply:
Anyway it would be nice to have it stressed in the test.